src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,100 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { request } from '../api/request';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+interface Props {
+  callbackForMovies: (arg0 : Movie[]) => void;
+  moviesPrev: Movie[];
+}
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = (
+  { callbackForMovies, moviesPrev },
+) => {
+  const [movieForFind, setMovieForFind] = useState('');
+  const [preview, setPreview] = useState<Movie>();
+  const [isFilmValid, setIsFilmValid] = useState(true);
+
+  async function finder() {
+    const result = await request(movieForFind.toLowerCase());
+
+    if (result.Response !== 'False') {
+      setPreview(result);
+    } else {
+      setMovieForFind('');
+      setPreview(undefined);
```

Comment:
It is better to set null instead of undefined

Reply:
why?

Reply:
because of difference between null and undefined
also, undefined doesn't exist in json format